r/R/csv.R

```diff
@@ -612,6 +622,8 @@ readr_to_csv_convert_options <- function(na,
 #' system (`SubTreeFileSystem`)
 #' @param include_header Whether to write an initial header line with column names
 #' @param batch_size Maximum number of rows processed at a time. Default is 1024.
+#' @param write_options read_options see [file reader options][CsvWriteOptions]
```

Comment:
The downside to this is that we do not have much in terms of content for `CsvWriteOptions`. We direct to it from a couple of places, but maybe we need to extent the documentation, more so once we surface more options from the C++ code.